Source/misdat.h

```diff
@@ -31,68 +31,68 @@ enum class DamageType : uint8_t {
 	Acid,
 };
 
-typedef enum missile_graphic_id : uint8_t {
-	MFILE_ARROWS,
-	MFILE_FIREBA,
-	MFILE_GUARD,
-	MFILE_LGHNING,
-	MFILE_FIREWAL,
-	MFILE_MAGBLOS,
-	MFILE_PORTAL,
-	MFILE_BLUEXFR,
-	MFILE_BLUEXBK,
-	MFILE_MANASHLD,
-	MFILE_BLOOD,
-	MFILE_BONE,
-	MFILE_METLHIT,
-	MFILE_FARROW,
-	MFILE_DOOM,
-	MFILE_0F,
-	MFILE_BLODBUR,
-	MFILE_NEWEXP,
-	MFILE_SHATTER1,
-	MFILE_BIGEXP,
-	MFILE_INFERNO,
-	MFILE_THINLGHT,
-	MFILE_FLARE,
-	MFILE_FLAREEXP,
-	MFILE_MAGBALL,
-	MFILE_KRULL,
-	MFILE_MINILTNG,
-	MFILE_HOLY,
-	MFILE_HOLYEXPL,
-	MFILE_LARROW,
-	MFILE_FIRARWEX,
-	MFILE_ACIDBF,
-	MFILE_ACIDSPLA,
-	MFILE_ACIDPUD,
-	MFILE_ETHRSHLD,
-	MFILE_FIRERUN,
-	MFILE_RESSUR1,
-	MFILE_SKLBALL,
-	MFILE_RPORTAL,
-	MFILE_FIREPLAR,
-	MFILE_SCUBMISB,
-	MFILE_SCBSEXPB,
-	MFILE_SCUBMISC,
-	MFILE_SCBSEXPC,
-	MFILE_SCUBMISD,
-	MFILE_SCBSEXPD,
-	MFILE_SPAWNS,
-	MFILE_REFLECT,
-	MFILE_LICH,
-	MFILE_MSBLA,
-	MFILE_NECROMORB,
-	MFILE_ARCHLICH,
-	MFILE_RUNE,
-	MFILE_EXYEL2,
-	MFILE_EXBL2,
-	MFILE_EXRED3,
-	MFILE_BONEDEMON,
-	MFILE_EXORA1,
-	MFILE_EXBL3,
-	MFILE_NONE,
-} missile_graphic_id;
+typedef enum class MissileGraphicID : uint8_t {
+	Arrow,
+	Fireball,
+	Guardian,
+	Lightning,
+	FireWall,
+	MagmaBallExplosion,
+	TownPortal,
+	FlashBottom,
+	FlashTop,
+	ManaShield,
+	BloodHit,
+	BoneHit,
+	MetalHit,
+	FireArrow,
+	DoomSerpents,
+	Golem,
+	Spurt,
+	ApocalypseBoom,
+	StoneCurseShatter,
+	BigExplosion,
+	Inferno,
+	ThinLightning,
+	BloodStar,
+	BloodStarExplosion,
+	MagmaBall,
+	Krull,
+	ChargedBolt,
+	HolyBolt,
+	HolyBoltExplosion,
+	LightningArrow,
+	FireExplosion,

```

Comment:
Suggestion to use `FireArrowExplosion` instead of `FireExplosion`, as both the file name `missiles/firarwex.cl2` and enum `MF_FAEXP` include *arrow*.